src/libraries/System.Private.Uri/src/System/Uri.cs

```diff
@@ -3314,19 +3315,7 @@ private unsafe void ParseRemaining()
                     origIdx = index == -1 ? _originalUnicodeString.Length : (index + origIdx);
                 }
 
-                // Correctly escape unescape
-                string escapedPath = EscapeUnescapeIri(_originalUnicodeString, offset, origIdx, UriComponents.Path);
-
-                // Normalize path
-                try
-                {
-                    _string += escapedPath;
-                }
-                catch (ArgumentException)
-                {
-                    UriFormatException e = GetException(ParsingError.BadFormat)!;
-                    throw e;
-                }
+                _string += EscapeUnescapeIri(_originalUnicodeString, offset, origIdx, UriComponents.Path);
```

Comment:
The try-catch block removal is, I presume, due to `+=` never actually throwing?

Reply:
Exactly, it's just dead code